types/api/v1/cluster_types.go

```diff
@@ -59,6 +59,7 @@ type ClusterSpec struct {
 	CertSANS []string `json:"certSANS,omitempty"`
 	Masters  Hosts    `json:"masters,omitempty"`
 	Nodes    Hosts    `json:"nodes,omitempty"`
+	Registry string   `json:"registry,omitempty"`
```

Comment:
Clusterfile adding fields requires review, and making as few changes as possible.